app/book_model.py

```diff
@@ -1,15 +1,72 @@
-books = []
+library = []
 
 
 class Book:
 
-    def __init__(self, title, author, book_number, publisher):
+    def __init__(self, title, author, id, publisher, number_of_books):
         self.title = title
         self.author = author
-        self.book_number = book_number
+        self.id = id
         self.publisher = publisher
+        self.number_of_books = number_of_books
+
+    def delete_book(self, id):
+        for book in library:
+            if book[id] == id:
+                library.remove(book)
+
+        return "book removed successfully, {} books remaining".format(len(library))
+
+    def add_book(self, title, author, id, publisher, number_of_books):
```

Comment:
This way you are adding the book is not following the OOP standards. You are supposed to create an instance of the class when creating a new book.